terracumber_config/tf_files/SUSEManager-3.2-refenv-PRV.tf

```diff
@@ -118,7 +118,7 @@ module "suse-client" {
   base_configuration = module.base.configuration
   product_version    = "3.2-nightly"
   name               = "cli-sles12"
-  image              = "sles12sp4"
+  image              = "sles12sp4o"
```

Comment:
If you say we changed the default images at https://github.com/uyuni-project/sumaform/pull/741, why do we need to specify the images here? (same for all other places)

Reply:
You are right.  We don't need tuch an eager commit; we can keep the `image` variables in those files.
Then, the only we need to do here is to update the variable `images`.
- it must contain defaults as centos7o or sles15sp1o in those main.tf where the default is implied
(e.g.  `SUSEManager-4.1-PRV.tf`).

Reply:
Now, `image` is only specified when it sets a value other than the default.